doc/man3/X509_LOOKUP_meth_new.pod

```diff
@@ -150,10 +150,20 @@ the X509_LOOKUP context, the type of the X509_OBJECT being requested, parameters
 related to the lookup, and an X509_OBJECT that will receive the requested
 object.
 
-Implementations should use either X509_OBJECT_set1_X509() or
-X509_OBJECT_set1_X509_CRL() to set the result. Any method data that was
-created as a result of the new_item function set by
-X509_LOOKUP_meth_set_new_item() can be accessed with
+Implementations must add objects it finds to the B<X509_STORE> object
+using X509_STORE_add_cert() or X509_STORE_add_crl().  This "duplicates"
+the object by incrementing its reference counter.  However, the
```

Comment:
IMO "duplicates" is not a good phrase here, because it is a synonym of "copies". (I know, that's why you put the double quotes around it.)  How about the following alternative?

```
Implementations must add objects they find to the
B<X509_STORE> object using X509_STORE_add_cert()
or X509_STORE_add_crl().  Note that in doing so,
the reference counter of the object is incremented.
```

Reply:
Okie